### ISSUE TITLE
Entry Point Not Found - The procedure entry point for_realloc_lhs could not be located in the dynamic link library 

### ISSUE THREAD

**Issue by stephen0405** [OpenFAST/openfast#775]
Hi,

I'm quite new to OpenFAST and will be using it over the coming months for a structural research project I'm carrying out.

I have gotten an 'Entry Point Not Found' error when I have tried to launch it from the command prompt (See attached image).

It would appear that others have had similar problems in the past, see links below:
https://github.com/OpenFAST/openfast/issues/649
https://github.com/OpenFAST/openfast/issues/697

However I have not found the exact same error (with 'for_realloc_lhs') encountered anywhere before.

I have tried most of the suggested solutions here and on youtube too, such as downloading the latest versions of the Windows c++ redistributable package. But nothing I have done so far has fixed the issue.

Does anyone have any other suggestions on how to solve this problem?
Any help would be greatly appreciated.

![Error_1full](https://user-images.githubusercontent.com/85649287/121569944-4117bd00-ca19-11eb-9425-661a849c2c8f.PNG)

Kind regards,
Stephen Lynch

**Comment by rafmudaf** [comment]
Please provide information on the version of OpenFAST you're using and how you obtained it. A template of good information is below. You can display the binary's info by running it without any arguments.

**OpenFAST Version**
<Please provide as much detail as possible including git commit. The best information is a screenshot of the OpenFAST system description that prints when running OpenFAST:>

```
**************************************************************************************************
 OpenFAST

 Copyright (C)  National Renewable Energy Laboratory
 Copyright (C)  Envision Energy USA LTD

 This program is licensed under Apache License Version 2.0 and comes with ABSOLUTELY NO WARRANTY.
 See the "LICENSE" file distributed with this software for details.
 **************************************************************************************************

 OpenFAST-v2.0.0
 Compile Info:
  - Architecture: 64 bit
  - Precision: double
  - Date: Nov 27 2018
  - Time: 17:19:38
 Execution Info:
  - Date: 11/29/2018
  - Time: 10:52:28-0700
```

**System Information (please complete the following information):**
 - OS: <e.g. Ubuntu 14.04 or macOS 10.12>
 - Compiler: <e.g. GFortran 4.4>
 - Compiler settings: <e.g. CMake flags or other settings>

**Comment by rafmudaf** [comment]
@stephen0405 Did you find something that resolved this problem? If so, what was the solution? It's helpful for us to know for our and other user's future reference.

**Comment by stephen0405** [comment]
@rafmudaf Yes I have resolved the issue. 

This error appeared when I was trying to use an 'openfast_x64.exe' application which I didn't build myself, but instead was sent to me by a colleague. So I decided to go back and follow the directions in OpenFAST Documentation Release v2.5.0, section 2.2.3 Visual Studio Solution for Windows, and the executable developed works fine now without getting this error.

I did have some further issues with the fortran compiler when using Visual Studio, but I overcame these issues using information I found at the following links:
https://community.intel.com/t5/Intel-Fortran-Compiler/The-Fortran-compiler-ifort-exe-cannot-be-found/m-p/1254567
https://software.intel.com/content/www/us/en/develop/articles/intel-compilers-compatibility-with-microsoft-visual-studio-and-xcode.html


Thanks for offering your help anyway.
